tests/unit/cli/admin/test_ceremony.py

```diff
@@ -312,11 +347,20 @@ def test_ceremony_online_key_one_of_root_keys(
         # Test that online key cannot be one of root key's.
         input_step1, input_step2, _, input_step4 = ceremony_inputs
         input_step3 = [  # Configure Online Key
-            f"{_PEMS / 'JH.pub'}",  # Please enter path to public key
-            f"{_PEMS / '0d9d3d4bad91c455bc03921daa95774576b86625ac45570d0cac025b08e65043.pub'}",  # Please enter path to public key  # noqa
             "Online Key",  # Please enter a key name
         ]
 
+        selection = iter(
+            (
+                f"{_PEMS / 'JH.pub'}",  # Please enter path to public key
+                f"{_PEMS / '0d9d3d4bad91c455bc03921daa95774576b86625ac45570d0cac025b08e65043.pub'}",  # Please enter path to public key  # noqa
+            )
+        )
```

Comment:
The problem is that you need the first prompt_toolkit to input the root keys. You mocked only the iterations for the online key.

```suggestion
        selection = iter(
            (
                f"{_PEMS / 'JC.pub'}",   # Root key 1
                f"{_PEMS / 'JH.pub'}",   # Root key 2
                f"{_PEMS / 'JJ.pub'}",    # Root key 3           
                f"{_PEMS / 'JH.pub'}",  # Please enter path to public key
                f"{_PEMS / '0d9d3d4bad91c455bc03921daa95774576b86625ac45570d0cac025b08e65043.pub'}",  # Please enter path to public key  # noqa
            )
        )
```

Reply:
But where are the inputs to those keys coming from? It's not obvious to me.